Xamarin.Forms.Platform.Android/Renderers/BorderDrawable.cs

```diff
@@ -221,25 +228,21 @@ public RectF GetPaddingBounds(int width, int height)
 
 		public void DrawOutline(Canvas canvas, int width, int height)
 		{
-			if (BorderElement.BorderWidth <= 0)
-				return;
-
 			using (var paint = new Paint { AntiAlias = true })
 			using (var path = new Path())
 			{
 				float borderWidth = _convertToPixels(BorderElement.BorderWidth);
-				float inset = borderWidth / 2;
 
 				// adjust border radius so outer edge of stroke is same radius as border radius of background
-				float borderRadius = Math.Max(ConvertCornerRadiusToPixels() - inset, 0);
+				float borderRadius = Math.Max(ConvertCornerRadiusToPixels(), 0);
 
 				RectF rect = new RectF(0, 0, width, height);
-				rect.Inset(inset + PaddingLeft, inset + PaddingTop);
```

Comment:
@samhouts after some testing , adding this back and subtracting the inset above that we remove, fix the issue with the corner radius that I noticed in my review, at the same time it makes the border width look bigger

Reply:
I'm seeing the same thing as @rmarinho  on this one
I've been playing with this for a little bit and here's where I am so far

https://gist.github.com/PureWeen/855e7acdffe2087553a75881f95d23f6
Basically I changed the insets a bit because I wasn't quite sure on the intent of the math in the PR

I inset the background just based on padding and shadow
And then I inset the Outline the same but then plus half the borderwidth which just shrinks it down so the outer edge of the stroke lines up with the background

The effect of this is that if you set a really wide stroke you'll still see a shadow